docs/reference/isr_rules.rst

```diff
@@ -124,6 +124,30 @@ A means of creating an object without employing a class or globals is as follows
 The compiler instantiates the default ``buf`` argument when the function is
 loaded for the first time (usually when the module it's in is imported).
 
+An instance of object creation occurs when a reference to a bound method is created. This means that
```

Comment:
Can you please wrap lines similarly to the paragraph above. (I also would recommend wrapping them at <80 chars for any new paragraphs).